# General Parameters
odom_frame: "robot1/odom"
map_frame: "robot1/map"
base_frame: "robot1/base_link"

# topic 
pub_submap_name: "robot1/map"
sub_odom_name: "robot1/odom"
sub_scan_name: "robot1/scan"

# mapper
minimum_time_interval: 3600
minimum_travel_distance: 0.1 
minimum_travel_heading: 0.1                  #in radians
use_scan_matching: false
do_loop_closing: false
scan_buffer_size: 10
scan_buffer_maximum_scan_distance: 100
# enternal
map_update_interval: 0
use_scan_range: 8
resolution: 0.05

max_queue_size: 10
now_to_last_dist: 1.0

use_real_param: false